stores/evga.py

```diff
@@ -0,0 +1,169 @@
+                    username = config["username"]
+                    password = config["password"]
+                    self.credit_card["name"] = config["credit_card"]["name"]
+                    self.credit_card["number"] = config["credit_card"]["number"]
+                    self.credit_card["cvv"] = config["credit_card"]["cvv"]
+                    self.credit_card["expiration_month"] = config["credit_card"][
+                        "expiration_month"
+                    ]
+                    self.credit_card["expiration_year"] = config["credit_card"][
+                        "expiration_year"
+                    ]
+        except Exception as e:
+            log.error(f"This is most likely an error with your {CONFIG_PATH} file.")
+            raise e
+
+        self.login(username, password)
+
+    def login(self, username, password):
+        """
+        We're just going to enter the user info and let the user handle the captcha
+        :param username:
+        :param password:
+        :return:
+        """
+        self.driver.execute_cdp_cmd(
+            "Network.setUserAgentOverride",
+            {
+                "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.53 Safari/537.36"
+            },
+        )
+        self.driver.execute_script(
+            "Object.defineProperty(navigator, 'webdriver', {get: () => undefined})"
+        )
+
+        self.driver.get(LOGIN_URL)
+        selenium_utils.wait_for_page(self.driver, "EVGA - Intelligent Innovation")
+
+        selenium_utils.field_send_keys(self.driver, "evga_login", username)
+        selenium_utils.field_send_keys(self.driver, "password", password)
+
+        log.info("Go do the captcha and log in")
+
+        selenium_utils.wait_for_page(
+            self.driver, "EVGA - Intelligent Innovation - Official Website", 300
+        )
+        log.info("Logged in!")
+
+    def buy(self, delay=5):
+        self.driver.get(
+            "https://www.evga.com/products/productlist.aspx?type=0&family=GeForce+30+Series+Family&chipset=RTX+3080"
+        )
+        # self.driver.get("https://www.evga.com/products/ProductList.aspx?type=0&family=GeForce+16+Series+Family&chipset=GTX+1660")
+
+        selenium_utils.wait_for_page(
+            self.driver,
+            "EVGA - Products - Graphics - GeForce 30 Series Family - RTX 3080",
+        )
+        # selenium_utils.wait_for_page(self.driver, "EVGA - Products - Graphics - GeForce 16 Series Family - GTX 1660")
+
+        #  Check for stock
+        log.info("On GPU Page")
+        atc_buttons = self.driver.find_elements_by_xpath(
+            '//input[@class="btnBigAddCart"]'
+        )
+        while not atc_buttons:
+            log.debug("Refreshing page for GPU")
+            self.driver.refresh()
+            atc_buttons = self.driver.find_elements_by_xpath(
+                '//input[@class="btnBigAddCart"]'
+            )
+            sleep(delay)
```

Comment:
Delay not getting passed in. sleep(5) works.

Reply:
Fixed.